### ISSUE TITLE
_hasHistoryAPI

### ISSUE THREAD

**Issue by favrio** [cssmagic/history.state#13]
``` javascript
function _hasHistoryAPI() {
        return 'onpopstate' in window &&
            'replaceState' in history && typeof history.replaceState === 'function' &&
            'replaceState' in history && typeof history.replaceState === 'function'
    }
```

应该是检测pushState 和 'replaceState' 吧，代码里面2个'replaceState'

**Comment by cssmagic** [comment]
谢谢反馈！此系笔误。

**Comment by cssmagic** [comment]
Closed by PR #14.
